bless/config/bless_deploy_example.cfg

```diff
@@ -1,13 +1,15 @@
 # This section and its options are optional
 [Bless Options]
 # Number of seconds +/- the issued time for the certificate to be valid
-certificate_validity_window_seconds = 120
+certificate_validity_seconds = 120
```

Comment:
I thought "validity window" was a useful config for short lived client certs.  However for host certs, this seems like it wouldn't really be ideal for your use case, assuming you issue host certs for far off valid_before range.

Would separate valid_before and valid_after offsets better suit your needs?  I'm guessing you don't have a need to issue valid after t=0 certs.